scripts/scil_compute_hdf5_average_density_map.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Compute a density map for each edges of a connection from a HDF5 file.
+Typically use after scil_decompose_connectivity.py in order to obtain the
+average density map of each connections to enable the use of --similarity
+in scil_compute_connectivity.py.
```

Comment:
Honestly, after reading this small paragraph, I'm not sure what this script is about to do... Maybe lets discuss in the slack...

Reply:
Just read input arguments needed and still not sure... 

Maybe give a simple example to illustrate what it does?

Reply:
connections -> connection